lib/msal-browser/src/utils/MathUtils.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Utility class for math specific functions
+ */
+export class MathUtils {
```

Comment:
I know we've bit nitpicking names a lot here, but I don't feel like `MathUtils` is really great either. I think this can probably get tossed in one of the other util files. Could also consider importing a library that does this function for us

Reply:
I disagree about the MathUtils, but we can figure out naming in the future. I feel like we have been trying to go with keeping files small and targeted, so I'm kind of getting mixed signals here in terms of code design. While I am all for importing other libraries, it is important we take into consideration security and performance.

Reply:
Sorry to confuse--yes, I agree smaller files are almost always better than shoving things into a big file. I don't necessarily have a great suggestion for the name of this file against `MathUtils`, which is part of why I suggested it may belong in one of the other files. I'm not too opinionated on what happens here, just was leaving some thoughts. I think this is fine if it's what we're working with.